src/components/strategies/marketPriceIndication/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { FC } from 'react';
+import BigNumber from 'bignumber.js';
+import { Tooltip } from 'components/common/tooltip/Tooltip';
+
+type Props = {
+  marketPricePercentage: BigNumber;
+  isRange?: boolean;
+};
+
+export const MarketPriceIndication: FC<Props> = ({
```

Comment:
why suddenly use index.ts for the component?

Usually our approach is that we have an index file with exports incase we have a folder with multiples files. The index file will indicate which files are exposed outside.

Reply:
Fixed that. Create index.ts file to export all of the needed component,hooks,types. Created also MarketPriceIndication.tsx to contain the component  & now the hook is inside that folder.